mods/cnc/rules/vehicles.yaml

```diff
@@ -154,7 +154,7 @@ ARTY:
 	Explodes:
 		Weapon: UnitExplode
```

Comment:
Aye this is true. I will need to do that. I will copy paste the command line down incase a future issue arises.